src/netex-convertor/period-ticket/periodTicketNetexHelpers.ts

```diff
@@ -1,17 +1,97 @@
-import { Stop } from '../types';
+import { Stop, OperatorData, PeriodGeoZoneTicket, PeriodMultipleServicesTicket } from '../types';
+import { NetexObject, getCleanWebsite } from '../sharedHelpers';
 
-export interface NetexObject {
-    [key: string]: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+export const getScheduledStopPointsList = (stops: Stop[]): {}[] =>
```

Comment:
Would it make sense to have a type on this, rather than an array of objects? Something like:

```
export interface ScheduledStopPoints {
versionRef: string;
ref: string;
$t: string;
}
``` ?

Reply:
Introduced a type for the return.